gensim/models/doc2vec_inner.c

```diff
@@ -859,7 +859,7 @@ static const char *__pyx_f[] = {
 #endif
 
 
-/* "../../.virtualenvs/release/local/lib/python2.7/site-packages/Cython/Includes/numpy/__init__.pxd":776
+/* "../../envs/gensim/lib/python3.7/site-packages/Cython/Includes/numpy/__init__.pxd":776
```

Comment:
revert back all unrelated `*.c` please (only `_utils_any2vec.c` needed in current PR)